public/app/plugins/datasource/loki/components/LokiQueryFieldForm.tsx

```diff
@@ -70,6 +70,7 @@ export interface LokiQueryFieldFormProps extends ExploreQueryFieldProps<DataSour
   syntax: any;
   logLabelOptions: any[];
   syntaxLoaded: any;
+  range: TimeRange;
```

Comment:
If we are to add this to the query editor interface it should be added to the base QueryEditorProps interface (and property should be named timeRange so it's named the same as in PanelProps interface)

Reply:
@torkelo we're using a property named `absoluteRange` of type `AbsoluteRange` now. Still feel this should go into `QueryEditorProps` interface?

Reply:
maybe it's not needed, the range is also available on the last query request/response

Reply:
Okay so let's move on with this and we may look further into this later.